Assets/HoloToolkit/Input/Scripts/CursorManager.cs

```diff
@@ -1,65 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I want us to be careful with these statements. Let's have an inclusive mindset where we respect some people find value for these type of classes. I agree with the sentiment however that is a blanket argument. I will add it back if I see a need for it.